MediaPlayer.py

```diff
@@ -152,12 +152,16 @@ def read_playlists(self):
 
 	def read_track_metadata(self, track):
 		tag = AudioTag(track.Artist.Name, track.Album.Name, track.Title)
-		tag.rating = self.get_normed_rating(track.Rating)
+		if track.Rating <0: 
+			tag.rating=0
+		else:
+			tag.rating = self.get_normed_rating(track.Rating)
```

Comment:
sharing this logic with all consumers of `get_normed_rating` might make sense. Did you consider this? I.e. putting this condition inside the function implementation.